index.bs

```diff
@@ -9725,6 +9725,31 @@ corresponding to [=interface members=].
 </div>
 
 
+<h4 id="Historical" extended-attribute lt="Historical">[Historical]</h4>
+
+When the [{{Historical}}] [=extended attribute=] appears on an individual
+[=dictionary member=], [=interface member=], [=interface mixin member=], or
+[=namespace member=], it indicates that the construct exists for historical
+reason.
+
+The [{{Historical}}] extended attribute must
+[=takes no arguments|take no arguments=].
+
+<div class="example">
+    There are Web APIs that have been replaced by better new APIs and are
```

Comment:
Generally we keep web lowercase as per https://whatwg.org/style-guide (unless it's part of a title or some such) though it seems IDL is somewhat inconsistent on this.

Reply:
Yes, quite a lot `Web` are in this spec, probably time to fix them.